npm/private/npm_import.bzl

```diff
@@ -452,7 +452,7 @@ def _download_and_extract_archive(rctx, package_json_only):
 
     # npm packages are always published with one top-level directory inside the tarball, tho the name is not predictable
     # so we use tar here which takes a --strip-components N argument instead of rctx.download_and_extract
-    tar_args = ["tar", "-xf", _TARBALL_FILENAME, "--strip-components", "1", "-C", _EXTRACT_TO_DIRNAME, "--no-same-owner", "--no-same-permissions"]
+    tar_args = ["tar", "--extract", "--file", _TARBALL_FILENAME, "--strip-components", "1", "--directory", _EXTRACT_TO_DIRNAME, "--no-same-owner", "--no-same-permissions"]
```

Comment:
Is this change necessary? This might be causing the windows e2e/bzlmod failure:
```
tar.exe: Error opening archive: Can't initialize filter; unable to run program "gzip -d"
```

Reply:
It's not necessary - I just wanted to use the long form as a matter of style. Will put that in a separate change.

Reply:
I agree with that, but it broke windows...

Reply:
reverted. CI is still red, but so is `main`. Want me to force-merge?

Reply:
Just confirm it's only the auth issues first. With this tar args change windows was failing with a tar error